src/components/DigitalConverterComponent.cpp

```diff
@@ -0,0 +1,119 @@
+#include <iostream>
+
+#include <hyro/factory/CommandFactory.h>
+#include <hyro/DigitalConverterComponent.h> 
+
+using namespace std::chrono_literals;
+
+namespace hyro
+{
+hyro::Signal message_signal_rec;
+hyro::Signal message_output;
+Thresholding thold;
```

Comment:
This variables shall be a member class variable.

Please, do not use global variables.

Reply:
The variable `Thresholding thold` is used in different methods, such as the getters and setters. Should I declare this variable inside each function, every time I call one of these methods?